medicai/healthRecord/routes.py

```diff
@@ -0,0 +1,56 @@
+from flask import request, jsonify
+from flask_jwt_extended import create_access_token, jwt_required, get_jwt_identity
+from flask_cors import CORS, cross_origin
+from medicai.models.healthRecord import HealthRecord
+from medicai.extensions import db
+from medicai.extensions import bcrypt
+from medicai.healthRecord import bp
+
+
+@bp.route('/register', methods=['POST'])
+@cross_origin()
+def register():
+    data = request.get_json()
+    if 'userId' not in data:
+        return jsonify(message="User ID is required"), 400
+    
+    user_exists = HealthRecord.query.filter_by(userId=data['userId']).first() is not None
```

Comment:
existing_user, not user_exists

Reply:
you should check if the list is not empty and not None before accessing first() method